pkgs/by-name/mj/mjmap/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromSourcehut,
+  unstableGitUpdater,
+}:
+
+buildGoModule {
+  pname = "mjmap";
+  version = "0.1.0-unstable-2023-11-13";
+
+  src = fetchFromSourcehut {
+    owner = "~rockorager";
+    repo = "mjmap";
+    rev = "d54badae8152b4db6eec8b03a7bd7c5ff1724aa7";
+    hash = "sha256-yFYYnklNNOHTfoT54kOIVoM4t282/0Ir4l72GmqlGSY=";
+  };
+
+  vendorHash = "sha256-fJuPrzjRH0FpYj2D9CsFdsdzYT0C3/D2PhmJIZTsgfQ=";
+
+  installCheckPhase = ''
+    runHook preInstallCheck
+
+    $out/bin/mjmap --version >/dev/null
+
+    runHook postInstallCheck
+  '';
```

Comment:
Perhaps this is best expressed through a `versionCheckHook` or `passthru.tests.version = testVersion`?

Reply:
Nope, because it reports its versions as `0.0.0` 🫠

Reply:
Nice.

Reply:
I could add an explanatory comment, though.